mods/common/chrome/connection.yaml

```diff
@@ -25,7 +25,7 @@ Background@CONNECTIONFAILED_PANEL:
 			Y: 76
 			Width: PARENT_RIGHT
 			Height: 25
-			Text: ServerError
+			Text: Server Error
```

Comment:
Is this text even used? Better to remove it if not.

Reply:
It's not used. It's always overwritten on widget creation